providers/ms365/provider/provider.go

```diff
@@ -128,7 +128,6 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 	asset.Connections[0].Id = conn.ID()
 	s.runtimes[conn.ID()] = &plugin.Runtime{
 		Connection:     conn,
-		Resources:      syncx.Map[string]plugin.Resource{},
```

Comment:
not necessary anymore for the `syncx.Map`, since it uses `sync.Map` under the hood